# Request
---
# Response
bool setsave_flash_accepted
